exchange/exchange.go

```diff
@@ -1280,6 +1280,20 @@ func listBiddersWithRequests(bidderRequests []BidderRequest) []openrtb_ext.Bidde
 	return liveAdapters
 }
 
+func getMediaTypeForBid(bid openrtb2.Bid) (openrtb_ext.BidType, error) {
```

Comment:
Nitpick: Consider changing the name to `getPrebidMediaTypeForBid` to clarify what field this method is extracting.

Reply:
Done!